quantum/quantum.h

```diff
@@ -16,32 +16,33 @@
 #ifndef QUANTUM_H
 #define QUANTUM_H
 
-#if defined(__AVR__)
+#ifdef __AVR__
```

Comment:
could this be rolled back? 
```suggestion
#if defined(__AVR__)	
```

Easier to modify, if we need to add other checks.

Reply:
I don't think that is a likely scenario. Besides, there are many similar checks throughout the codebase to which similar logic could be applied, and I don't think changing it is worth it. I was going with “use the long form only if it's required (e.g. in `#elif`) or if an adjoining statement already uses the long form”.

Reply:
But if you insist, I can change it back.

Reply:
As a project keep going back and forth on this pattern. Last I knew we were preferring the long form for consistency's sake.

Reply:
If you'd like, I can change all `#ifdef`s I modified in this PR to the long form, for consistency.

Reply:
Though I believe it's consistent enough as it currently stands.

Reply:
Motivated by the rationale presented in #5500, I've decided to roll this change back.